test/src/test/resources/fixtures/instrumentation/execution-is-aborted-when-begin-execute-completes-exceptionally-inside-cf.yml

```diff
@@ -0,0 +1,66 @@
+name: execution is aborted when beginExecute completes exceptionally inside cf
+enabled:
+  current: true
+  nextgen: true
+overallSchema:
+  MyService: |
+    service MyService {
+      type Query {
+        hello: World
+      }
+      type World {
+        id: ID
+        name: String
+      }
+      type Mutation {
+        hello: String
+      }
+      type Subscription {
```

Comment:
this `Subscription` type doesn't seem to be relevant to test and can be a bit distracting.

Reply:
Ah yea, it's a copy paste from another test. I'll remove it.